utils/warnings.ml

```diff
@@ -555,40 +555,40 @@ let descriptions = [
                    (struct end) rather than to ().";
     since = since 5 1 };
   { number = 101;
-    names = ["unused-bs-attributes"];
-    description = "Unused bs attributes";
+    names = ["unused-mel-attributes"];
+    description = "Unused mel attributes";
     since = None };
 
   { number = 102;
     names = ["polymorphic-comparison-introduced"];
     description = "Polymorphic comparison introduced (maybe unsafe)";
     since = None };
   { number = 103;
-    names = [ "bucklescript-ffi-warning" ];
+    names = [ "melange-ffi-warning" ];
     description = "Fragile FFI definitions";
     since = None };
   { number = 104;
-    names = [ "bucklescript-bs-deriving" ];
-    description = "bs.deriving warning with customized message ";
+    names = [ "melange-mel-deriving" ];
+    description = "mel.deriving warning with customized message ";
```

Comment:
`mel.deriving` doesn't exist. maybe we should retire this warning. I'm not sure if we emit it anymore.

Reply:
It is used in [unrecognizedConfigRecord](https://github.com/melange-re/melange/blob/1167ca745c7ddc2b950559e53d2ebe43585f3850/jscomp/common/ast_payload.ml#L38).

I managed to find a couple of cases where they appear, but don't ask me _why_ `bs.deriving` was mentioned: 😄 

```
1 | [@@@bs.config { foo with bar = 2 }]
                  ^^^^^^^^^^^^^^^^^^^^
Warning 104 [bucklescript-bs-deriving]: bs.deriving warning: `with` is not supported, discarding
```

Another:

```
2 | [@mel.config [1, 2, 3]];
      ^^^^^^^^^^
Warning 104 [bucklescript-bs-deriving]: bs.deriving warning: invalid attribute config-record, ignoring
```

Maybe we should rename it to `Warnings.Mel_invalid_config`? Also, should that be done in this PR of a new one?

Reply:
It was a bs.deriving error because `bs.deriving` / `deriving` takes a record with configuration e.g. `[@@deriving abstract {light = light}]`.

I do think we should rename this to `Mel_invalid_config` or something. My preference is a new PR.

Reply:
Sounds good. Is this good to merge in that case?

Reply:
I think so. Let's do it!